libraries/freertos_plus/standard/freertos_plus_cli/source/FreeRTOS_CLI_Console.c

```diff
@@ -107,12 +102,14 @@ void FreeRTOS_CLIEnterConsoleLoop( xConsoleIO_t consoleIO,
     }
 }
 
-static void processInputBuffer( xConsoleIO_t consoleIO,
-                                int32_t inputSize,
-                                char * pCommandBuffer,
-                                size_t commandBufferLength,
-                                char * pOutputBuffer,
-                                size_t outpuBufferLength )
+void FreeRTOS_CLI_ProcessInputBuffer( xConsoleIO_t consoleIO,
```

Comment:
This duplicates some code in `FreeRTOS_CLI_ProcessInputBuffer` to handle the exiting the CLI loop. Can we keep code same and instead add a new command such as `exit` for the loop to exit gracefully.

Reply:
We discussed offline and decided that we can add a `exit` command support to the `FreeRTOS_CLIEnterConsoleLoop` function to allow the console loop to be terminated in a separate PR.